onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -2014,15 +2014,16 @@ static constexpr OrtApi ort_api_1_to_5 = {
     &OrtApis::SetGlobalIntraOpNumThreads,
     &OrtApis::SetGlobalInterOpNumThreads,
     &OrtApis::SetGlobalSpinControl,
+    &OrtApis::AddInitializer,
 };
```

Comment:
nit: AddInitializer will be in v6 won't it? So we need to add `// Version 6 - In development, feel free to add/remove/rearrange here` and update the version 5 section to say it should not be modified?